cmd/skopeo/inspect.go

```diff
@@ -187,10 +201,31 @@ func (opts *inspectOptions) run(args []string, stdout io.Writer) (retErr error)
 			logrus.Warnf("Registry disallows tag list retrieval; skipping")
 		}
 	}
-	out, err := json.MarshalIndent(outputData, "", "    ")
-	if err != nil {
+	if len(opts.format) == 0 {
+		out, err := json.MarshalIndent(outputData, "", "    ")
+		if err == nil {
+			fmt.Fprintf(stdout, "%s\n", string(out))
+		}
 		return err
 	}
-	fmt.Fprintf(stdout, "%s\n", string(out))
-	return nil
+	return outputFormat(outputData, opts.format)
+}
+
+func outputFormat(data interface{}, format string) error {
+	var out formats.Writer
+	if format == "json" || format == "" { // "" for backwards compat
+		out = formats.JSONStruct{Output: data}
+	} else {
+		out = formats.StdoutTemplate{Output: data, Template: inspectFormat(format)}
+	}
+	return out.Out()
```

Comment:
What does the `formats.` package do? AFAICT the only non-trivial effect is the `humanNewLine` call. Is that useful enough to add this dependency, complicating mass rebases? (Alternatively, moving the subpackage to c/common would mean no new dependencies are introduced.)

Reply:
https://github.com/containers/common/pull/320